landing_page/app/javascript/components/Home.test.js

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { render, unmountComponentAtNode } from "react-dom";
+import { act } from "react-dom/test-utils";
+import Home from './Home';
+
+let container = null;
+beforeEach(() => {
+  container = document.createElement("div");
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  unmountComponentAtNode(container);
+  container.remove();
+  container = null;
+});
+
+it( 'renders with a title', ()=>{
+    act( ()=>{render(<Home title='Welcome to the Blogging Engine'/>, container)} )
+        expect(container.textContent).toBe("Welcome to the Blogging Engine")
+});
+
+it( "doesn't render without a title", ()=>{
```

Comment:
Not sure I see the point of this test? I believe this is covered in the above test already where you test that whatever you put in the property title is rendered as text on the screen. This test merely changes what is rendered, in this case a blank string

Reply:
I'll remove those now 👍